_config.yml

```diff
@@ -23,12 +23,13 @@ description: "An open source Python-3 (CPython >= 3.5.0) Interpreter written in
 # baseurl: "/" # the subpath of your site, e.g. /blog
 url: "https://rustpython.github.io" # the base hostname & protocol for your site, e.g. http://example.com
 github_username:  RustPython
-disclaimer: "The disclaimer not to use this in production. Update this in the config.yml file at the root."
+disclaimer: "RustPython is in a development phase and should not be used in production or a fault intolerant setting. Our current build supports only a subset of Python syntax."
```

Comment:
do we know how much of the python syntax is supported? any ballpark? we can update this only when we reach milestones like 25%, 50%, 75% etc...

Reply:
We support pretty much all of the syntax at this point, I think that disclaimer is about a year old now... we definitely don't support all Python library features, though, maybe only half of the standard library is supported? I think that would be a good statistic to have there.

Reply:
ha cool. I think it will be cool to mention that you support 100% of the python syntax but only some of the standard library. It will be nice to highlight both sides :) 

thanks for merging :)